ods_ci/utils/scripts/rosa/rosa.py

```diff
@@ -37,6 +37,15 @@ def set_rosa_version(self):
             f"awk '{{print $1}}' | grep -w '^{re.escape(version)}*' | head -n1"
         )
         latest_version = execute_command(latest_version_cmd)
+        # when rosa cli is not the latest one, we see WARN messages, and this is breaking the automations when
+        # picking up the retrieved version, so adding this regex commands to remove the warnings.
+        latest_version = re.sub(
+            r"WARN: The current version \(\d+\.\d+\.\d+\) is not up to date with latest released "
+            r"version \(\d+\.\d+\.\d+\).",
+            "",
+            latest_version,
+        )
+        latest_version = re.sub(r"WARN: It is recommended that you update to the latest version.\n", "", latest_version)
         self.rosa_version = latest_version.strip()
```

Comment:
maybe instead of this specific warning regexp we can do it easily like:
```suggestion
        self.rosa_version = latest_version.splitlines()[-1].strip()
```

Reply:
thanks @kobihk let me test that